aldryn_search/base.py

```diff
@@ -130,7 +138,30 @@ def get_description(self, obj):
         """
         return None
 
+    def check_for_app_config(self, obj):
```

Comment:
@chronossc please don't add this to the base. Instead add it to https://github.com/aldryn/aldryn-search/blob/master/aldryn_search/search_indexes.py this is the index that deals with cms pages